src/cpp/exe/benders_mpi/main.cpp

```diff
@@ -36,6 +36,19 @@ int main(int argc, char** argv)
 	// Read options, needed to have options.OUTPUTROOT
 	BendersOptions options(build_benders_options(argc, argv));
 
+    auto masterLogger = std::make_shared<xpansion::logger::Master>();
+    Logger loggerUser = std::make_shared<xpansion::logger::User>(std::cout);
+    std::string loggerFileName = options.OUTPUTROOT + PATH_SEPARATOR + "reportbendersmpi";
+    if (world.rank() == 0)
+    {
+        loggerFileName += "-" + std::to_string(world.rank());
+    }
+    Logger loggerFile = std::make_shared<xpansion::logger::UserFile>(loggerFileName);
+    masterLogger->addLogger(loggerUser);
+    masterLogger->addLogger(loggerFile);
+
+    Logger logger = masterLogger;
```

Comment:
This copy should be discussed later with @sgatto.
This doesn't block the merge so we can deliver 0.4.0 version